cmd/fluent-bit-watcher/main.go

```diff
@@ -155,17 +157,19 @@ func start() {
 }
 
 func wait() {
-
+	mutex.Lock()
```

Comment:
What is the reason for locking here？

Reply:
To protect the read of `cmd` value since it is accessed in the fs notify watcher goroutine as well, via `stop()`.